app/src/main/java/bubi/com/logintest/screentests/HomeScreenRobot.java

```diff
@@ -2,20 +2,27 @@
 
 public class HomeScreenRobot {
 
+    private String viewName;
+
     public void findViewByName(String viewName){
+        this.viewName = viewName;
         if(viewName!=null){
-            System.out.println(viewName+" found");
-
+            printMessage(this.viewName +" found");
```

Comment:
Please leave space after the + character

Reply:
Ok

Reply:
Thanks